packages/hash/api/src/graphql/resolvers/pages/accountPages.ts

```diff
@@ -7,10 +7,12 @@ export const accountPages: Resolver<
   {},
   GraphQLContext,
   QueryAccountPagesArgs
-> = async (_, { accountId }, { dataSources }) => {
+> = async (_, { accountId, archived }, { dataSources }) => {
   const pages = await Page.getAllPagesInAccount(dataSources.db, {
     accountId,
   });
 
-  return pages.map((page) => page.toGQLUnknownEntity());
+  return pages
+    .filter((page) => !!page.properties.archived === archived)
```

Comment:
Is this the best approach, or should we create another query that would allow us to filter by the `archived` property and let this one return all account pages?

Reply:
I think the filtering logic might be best placed into the `getAllPagesInAccount` model class method, so that it's reusable in other places in the codebase.

Reply:
Addressed in 404a72db293732f2dd33b03090804f7dedc3dff8